managed/src/main/java/com/yugabyte/yw/controllers/CertificateController.java

```diff
@@ -145,9 +167,38 @@ public Result get(UUID customerUUID, String label) {
     }
   }
 
+  public Result delete(UUID certificate) {
+	CertificateInfo cert = CertificateInfo.get(certificate);
+	if (cert == null) {
+		return ApiResponse.error(BAD_REQUEST, "Invalid certificate.");
+	}
+	List<Universe> universes = Universe.getAll();
+	for (Universe universe : universes) {
+		// Try catch block for the universes which does not have certificate attached to it
+		try {
+			Universe universe_obj = Universe.get(universe.universeUUID);
```

Comment:
But you already have the universe. So why lookup again?

Reply:
I was getting Null every time if I use `universe` but when I get the object like above, I am getting the correct `rootCA` . Not sure what was the issue even though both objects look the same.

Reply:
In short 
`universe_obj.getUniverseDetails().rootCA;`  giving correct rootCA
but 
`univers.getUniverseDetails().rootCA;`   giving Null every single time.